chrome/js/onClick.js

```diff
@@ -0,0 +1,8 @@
+
+window.addEventListener("click", function(event) {
+  console.log('were in the clikc WINDOW', event)
+});
+
+document.addEventListener("click", function(event) {
```

Comment:
Would we want the event listener in both window and document?

Reply:
Unsure, that's why I left both there (see top comment)